package.json

```diff
@@ -33,18 +33,25 @@
   "homepage": "https://github.com/Code-Forge-Net/Remix-Dev-Tools#readme",
   "readme": "https://github.com/Code-Forge-Net/Remix-Dev-Tools#readme",
   "scripts": {
-    "prepublish": "npm run build",
-    "remix": "cd ./src/remix-app-for-testing && npm run dev",
-    "setup": "npm install && cd ./src/remix-app-for-testing && npm install && cd ../.. && npm run link",
-    "dev": "run-p  tailwind remix",
+    "prepublishOnly": "npm run build",
+    "remix": "npm run dev -w remix-app-for-testing",
+    "dev": "npm-run-all -s build:dev -p remix vite",
+    "build:dev": "vite build -m development",
     "build": "vite build",
-    "postbuild": "node moveStylesheet.js",
-    "vite": "vite build --watch",
+    "vite": "vite build --watch -m development",
     "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "prettier": "prettier --check .",
     "preview": "vite preview",
-    "test": "vitest run --coverage",
-    "tailwind": "npx tailwindcss -i ./src/input.css -o ./src/public/stylesheet.css --watch",
-    "link": "npm link ./src/remix-app-for-testing/node_modules/@remix-run/react"
+    "test": "vitest run --coverage"
+  },
+  "workspaces": [
+    ".",
+    "src/remix-app-for-testing"
+  ],
+  "peerDependencies": {
+    "@remix-run/react": "^1.18.1",
```

Comment:
I think this can be bumped down to 1.15, the package should work with all versions above it because it requires useNavigate from @remix-run/react